governance/polls/Proposal - Prioritising RWA - September 21, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Should MakerDAO domain teams prioritise the onboarding of Real World Asset collateral in the short term? - September 21, 2020
```

Comment:
Can't have '?'s in the titles. Maybe something like 'Prioritising the onboarding of Real World Asset collateral in the short term - date'?